bundle/compliance/policy/aws_ec2/data_adapter.rego

```diff
@@ -22,3 +36,33 @@ nacl_ingresses = res {
 ingresses_with_all_ports_open = res {
 	res = [entry | entry := nacl_ingresses[_]; not entry.PortRange]
 }
+
+# Get all the IpRanges from security groups that has an open inbound for all ipv4 cidr notions
+all_ipv4(entries) = res {
```

Comment:
since the data_adapter has access to the input, you can skip the param and just directly access `input.resource.IpPermissions`. unless you're expecting different types in the future?

Reply:
then the function name can be more specific 
```suggestion
security_groups_ip_permissions_<this function filter>(entries) = res {
```